.devcontainer/devcontainer.json

```diff
@@ -28,7 +24,22 @@
 	// "remoteUser": "root"
 
 	// VSCode extensions
-	"extensions": [
-		"ms-python.python",
-	],
+	"customizations": {
+		"vscode": {
+			"settings": {
+        "python.pythonPath": "/workspace/jaxsim/.pixi/envs/test-cpu/bin/python",
```

Comment:
Nit: indentation

Reply:
Solved with rebase in https://github.com/ami-iit/jaxsim/pull/276/commits/d817f0c586a0c45b1955ddf856e66cd71278a7eb